commands/createfixpullrequests.go

```diff
@@ -29,43 +29,57 @@ var pythonPackageRegexSuffix = "\\s*(([\\=\\<\\>\\~]=)|([\\>\\<]))\\s*(\\.|\\d)*
 
 type CreateFixPullRequestsCmd struct {
 	mavenDepToPropertyMap map[string][]string
+	// dryRun used for testing purposes, mocking part of the git commands that requires networking
+	dryRun bool
+	// When dryRun is enabled, dryRunRepoPath specifies the repository local path to clone
+	dryRunRepoPath string
```

Comment:
Let's add the description of the above `` and `` properties to the same properties in `GitManager`.

Reply:
done